tools/trap/ramdumpParser.py

```diff
@@ -958,7 +957,6 @@ def usage():
 	print('Following options are available')
 	print('\t-r, --dump_file                 RAM/FLASH dump_file along with path')
 	print('\t-t, --log_file                  Enter Logfile which contains stackdump during assert')
-	print('\t-G, --gdb_path                  Enter gdb tool path')
 	print('\t-N, --nm_path                   Enter nm tool path')
 	print('\t-E, --readelf_path              Enter readelf tool path')
```

Comment:
I dont think we need to provide an option to enter the tool paths.
Please check.

Reply:
Removed these options